api/handlers/node.go

```diff
@@ -64,7 +92,73 @@ func (h *Node) Identity(w http.ResponseWriter, r *http.Request) error {
 }
 
 func (h *Node) Peers(w http.ResponseWriter, r *http.Request) error {
-	peers := h.Network.Peers()
+	prs := h.Network.Peers()
+	resp := h.peers(prs)
+	return api.Render(w, r, resp)
+}
+
+func (h *Node) Topics(w http.ResponseWriter, r *http.Request) error {
+	allpeers, peerbytpc := h.TopicIndex.PeersByTopic()
+	alland := AllPeersAndTopicsJSON{}
+	tpcs := []topicIndexJSON{}
+	for topic, peerz := range peerbytpc {
+		tpcs = append(tpcs, topicIndexJSON{TopicName: topic, Peers: peerz})
+	}
+	alland.AllPeers = allpeers
+	alland.PeersByTopic = tpcs
+
+	return api.Render(w, r, alland)
+}
+
+func (h *Node) Health(w http.ResponseWriter, r *http.Request) error {
+	ctx := context.Background()
+	resp := healthCheckJSON{}
+	// check ports being used
+	addrs := h.Network.ListenAddresses()
+	for _, addr := range addrs {
+		resp.LocalPortsListening = append(resp.LocalPortsListening, addr.String())
+	}
+	// check consensus node health
+	err := h.NodeProber.CheckBeaconNodeHealth(ctx)
+	if err == nil {
+		resp.BeaconConnectionHealthStatus = Healthy.String()
+	} else {
+		resp.BeaconConnectionHealthStatus = fmt.Sprintf("%s: %s", NotHealthy.String(), err.Error())
+	}
+	// check execution node health
+	err = h.NodeProber.CheckExecutionNodeHealth(ctx)
+	if err == nil {
+		resp.ExecutionConnectionHealthStatus = Healthy.String()
+	} else {
+		resp.ExecutionConnectionHealthStatus = fmt.Sprintf("%s: %s", NotHealthy.String(), err.Error())
+	}
+	// check event sync health
+	err = h.NodeProber.CheckEventSycNodeHealth(ctx)
+	if err != nil {
+		resp.EventSyncHealthStatus = Healthy.String()
+	} else {
+		resp.EventSyncHealthStatus = fmt.Sprintf("%s: %s", NotHealthy.String(), err.Error())
+	}
+	// check peers connection
+	var activePeerCount int
+	prs := h.Network.Peers()
+	for _, p := range h.peers(prs) {
+		if p.Connectedness == "Connected" {
+			activePeerCount++
+		}
+	}
+	switch count := activePeerCount; {
```

Comment:
I see both `count` and `activePeerCount` being used in the switch-statement, so I'm not sure why we need to create the `count` variable here